shared/teams/add-members-wizard/confirm.tsx

```diff
@@ -41,7 +44,8 @@ const AddMembersConfirm = () => {
   // TODO: consider useMemoing these
   const anyNonKeybase = addingMembers.some(m => m.assertion.includes('@'))
   const someKeybaseUsers = addingMembers.some(member => !member.assertion.includes('@'))
-  const onlyEmails = !addingMembers.some(member => !member.assertion.endsWith('@email'))
+  const onlyEmails =
+    addingMembers.length > 0 && addingMembers.every(member => member.assertion.endsWith('@email'))
```

Comment:
can this be left as it was? short circuits and it's confusing that `every` returns true for an empty array.

Reply:
previous code was returning true for empty list as well

Reply:
my initial fix was to use `every` and then I realized i need to check length anyway... I can do length check +  `some` if that's better

Reply:
aha ok. scratch my initial comment then, this is fine.